### DISCUSSION TITLE
Why is everything broken

### DISCUSSION THREAD

**Discussion by outbackdingo** [scality/Zenko#1390] [General]
document states

Go to https://github.com/Scality/Zenko/releases and download the latest stable version of Zenko.
Unzip or gunzip the file you just downloaded and change to the top-level (Zenko) directory.
Configure with options.yaml
Create an options.yaml file in Zenko/kubernetes/ to store deployment parameters. Enter the following parameters:

ingress:
  enabled: true
  annotations:
    nginx.ingress.kubernetes.io/proxy-body-size: 0
  hosts:
    - zenko.local

cloudserver:
  endpoint: "zenko.local"

however there is no kubernetes directory

❯ cd Zenko-2.1.4
❯ ls -al
total 36
drwxrwxr-x. 1 dingo dingo   248 Nov 12 09:33 .
drwx--x---+ 1 dingo dingo  4628 Nov 12 10:17 ..
-rw-rw-r--. 1 dingo dingo    16 Nov 12 02:29 .dockerignore
drwxrwxr-x. 1 dingo dingo   164 Nov 12 02:29 docs
-rw-rw-r--. 1 dingo dingo    24 Nov 12 02:29 .eslintrc
drwxrwxr-x. 1 dingo dingo    88 Nov 12 02:29 eve
drwxrwxr-x. 1 dingo dingo   100 Nov 12 02:29 .github
-rw-rw-r--. 1 dingo dingo   155 Nov 12 02:29 .gitignore
-rw-rw-r--. 1 dingo dingo 10753 Nov 12 02:29 LICENSE
-rw-rw-r--. 1 dingo dingo  2142 Nov 12 02:29 README.md
-rw-rw-r--. 1 dingo dingo   506 Nov 12 02:29 .readthedocs.yml
drwxrwxr-x. 1 dingo dingo   222 Nov 12 02:29 res
drwxrwxr-x. 1 dingo dingo    84 Nov 12 02:29 solution
drwxrwxr-x. 1 dingo dingo    58 Nov 12 02:29 solution-base
drwxrwxr-x. 1 dingo dingo   218 Nov 12 02:29 tests
-rw-rw-r--. 1 dingo dingo    76 Nov 12 02:29 VERSION

**Comment by rahulreddy** [comment]
Hello @outbackdingo 
Sorry the latest release switched to using operator framework for deployment. If you are using Zenko 1.x. I recommend using 1.2.3 release which uses helm charts for deployment.
We will have instructions soon on how to deploy Zenko 2.x.

**Reply by TOMCO984** [reply]
Hi @rahulreddy ,
We see updates on the zenko repo but actually the documentation is outdated and unusable and zenko is impossible to install due to huge lack of documentation on actual versions.
The project is dead ?

**Comment by jameskim0987** [comment]
@rahulreddy I started following the docs for installing Zenko and its required components few weeks ago, however, the docs are mostly out-of-date and the installation steps were scattered out and not very straight forward and I'm still struggling to install Zenko. 

I have a couple questions here: 
1. Is the Zenko repo being actively maintained? I've raised a couple of issues and discussions but I'm not getting any comments regarding those issues.
2. When are the docs going to be updated? 
3. When will videos for clean installing Zenko (and its components) be released? I'm trying to clean install everything locally on my Ubuntu 18.04 VM so I can eventually play around with Zenko Orbit.
4. I'd like to experiment with Zenko. Is Minikube not an option for K8? Is it a must to use MetalK8 to use Zenko locally?
5. What is the best point of contact to get help on installation and technical issues?
6. Will there be a Slack channel for Zenko?

**Comment by hawksight** [comment]
I came across this due to not finding the the chart that is mentioned in the instruction in the recent releases. But thats even more confusing as the "latest" docs state Zenko `1.2.1` (see screen) whereas the latest GitHub release is `2.4.8`?

![Screenshot from 2022-06-21 15-54-19](https://user-images.githubusercontent.com/11051419/174831393-286c7dbd-3362-442c-9afd-d0e3593aa643.png)

So this doesn't work as there is no chart.

```shell
helm install --name my-zenko -f options.yaml zenko
```

Only chart references I can find are to some sub charts in the release package.

Following @rahulreddy's advice it looks like that last release with a helm chart is [1.2.3](https://github.com/scality/Zenko/releases/tag/1.2.3) which has the chart as an artifact for those looking for it.
But 1.2.3 is a long way from 2.4.8 and seems like we would be missing a lot of content?

If your moving away from helm in v2, then does that mean people trying to upgrade in the future will have to do a fresh install in the new method? Or is it just a case that docs need updated with the latest 2.X install method?
